src/features/Status/StatusCTAContainer.tsx

```diff
@@ -60,39 +54,19 @@ class StatusPage extends React.Component<IStatusPageProps, IStatusPageState> {
 
   public render() {
     return (
-      <Flex flexDirection={'column'} alignItems={'center'}>
-        <Helmet>
-          <title>Home | {HACKATHON_NAME}</title>
-        </Helmet>
-        <Box style={{ marginTop: '6rem' }}>
-          {this.props.confirmed && this.props.account ? (
-            <div>
-              <H1
-                color={theme.colors.red}
-                display={'absolute'}
-                textAlign={'center'}
-                marginLeft={'0'}
-              >
-                Hey {this.props.account.firstName},
-              </H1>
-              <StatusHeader
-                status={this.state.status}
-                settings={this.state.settings}
-                onClickConfirm={this.confirmStatus}
-                onClickWithdraw={this.withdrawStatus}
-              />
-              <BackgroundImage
-                right={'0px'}
-                bottom={'0px'}
-                src={Background}
-                imgHeight={'87%'}
-              />
-            </div>
-          ) : (
+      <div style={{ flex: 1, display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', padding: '6px 30px 96px 30px' }}>
```

Comment:
Consider `<Flex>` instead of manual styling.

Reply:
Done